damnit/ctxsupport/ctxrunner.py

```diff
@@ -36,7 +36,7 @@
 log = logging.getLogger(__name__)
 
 THUMBNAIL_SIZE = 300 # px
-
+COMPRESSION_OPTS = {'compression': 'gzip', 'compression_opts': 1, 'fletcher32': True, 'shuffle': True}
```

Comment:
Is there a specific reason to use the fletcher32 checksum? We dropped it from the gain/mask datasets in corrected data back in 2021, because we wanted to bypass HDF5 for writing, and we were assured that the filesystem is doing checksums already, so there's no particular need to do them at the HDF5 chunk level.

Reply:
>  we were assured that the filesystem is doing checksums already

Then I'll remove that option. I just remembered that it was used in the calibration code.